README.md

```diff
@@ -3,37 +3,38 @@
     <h1>PreMiD Presences</h1>
 </div>
 
-This repository contains the source code of all presences that are available in PreMiD Store. If you would like to <strong>publish</strong> your presence, please feel free to open a <a href="https://github.com/PreMiD/Presences/pulls">pull request</a>.
+This repository contains the source code of all Presences that are available in [**PreMiD's Store**](https://premid.app/store). If you wish to <strong>publish</strong> your Presence, open a [**Pull Request**](https://github.com/PreMiD/Presences/pulls).
 
 <div align="left">
     <a target="_blank" href="https://discord.premid.app/" title="Join our Discord!">
-        <img  src="https://discordapp.com/api/guilds/493130730549805057/widget.png?style=banner2" height="76px" draggable="false" alt="Join our Discord!">
+        <img  src="https://discordapp.com/api/guilds/493130730549805057/widget.png?style=banner2" height="75px" draggable="false" alt="Join our Discord!">
     </a>
 </div>
 
-## Requirements to create a presence
-
-- You must have the [latest TypeScript compiler](https://www.typescriptlang.org/#download-links) version installed
+## Requirements to create a Presence
+- [**Git**](https://git-scm.com)
+- [**Node.js**](https://nodejs.org/)
+- [**NPM**](https://npmjs.org/) (Or any other package manager)
 - Basic knowlege of TypeScript
 
 ## Getting started
 
-We made a documentation to make your development process a little bit better and faster. You can see it [**here**](https://docs.premid.app/dev/presence).
+Please refer to our [**documentation**](https://docs.premid.app/dev/presence).
```

Comment:
1. The current meaning kinda doesn't make sense if you ask me as it doesn't make it easier, it kinda makes it possible to understand in the first place since we don't have it described on GitHub to begin with.
2. Yeah, I forgot that GitHub colors them.